src/components/head.js

```diff
@@ -23,10 +24,14 @@ export default (props) => {
     .filter(Boolean)
     .join(" | ")
   const description =
-    (frontmatter && frontmatter.description) || props.description
+    (frontmatter && frontmatter.description) || props.description || "Swizec helps you become a better frontend engineer with books, articles, talks, and workshops";
 
-  const image = props.image || getSocialCard(frontmatter)
-  const url = `${props.path}`
+  const socialImage = getSocialCard(frontmatter);
+  const image = `https://swizec.com${socialImage || defaultHero}`;
+  // const url = `${props.path}`
+  const url = `https://swizec.com${
+    props.pageName !== undefined ? `/${props.pageName}` : ""
```

Comment:
In what case would the pageName be undefined?

Reply:
I changed it to props.path. pageName was always undefined (so the url was always https://swizec.com/)